desktop/core/src/desktop/static/desktop/js/listdir-inline.js

```diff
@@ -383,7 +383,7 @@ var FileBrowserModel = function (files, page, breadcrumbs, currentDirPath) {
   });
 
   self.isTaskServerEnabled = ko.computed(function() {
-    return window.getLastKnownConfig().hue_config.enable_chunked_file_uploader && window.getLastKnownConfig().hue_config.enable_task_server;
+    return window.getLastKnownConfig().storage_browser.enable_chunked_file_upload && window.getLastKnownConfig().hue_config.enable_task_server;
```

Comment:
I think at some point we should load the config into a settings class/object that use camelCase instead of the python generated snake_case. Not for now, just informing :-)